bears/python/PyreBear.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='pyre',
+        output_format='regex',
+        output_regex=r'\s*(?P<filename>.+)\s*:(?P<line>\d+):(?P<column>\d+)'
+                     r' \s*(?P<message>.+)',
+        global_bear=True)
+class PyreBear:
+    """
+    Checks Python files for errors using ``pyre``.
+    Run bear with `` --files=./<MY_DIR>/*``.
+
+    See https://pyre-check.org/ for more info.
+    """
+    LANGUAGES = {'Python', 'Python 3'}
+    REQUIREMENTS = {PipRequirement('pyre', '0.0.11')}
```

Comment:
increase the minimum version.
nice work on reporting the bug upstream.

Reply:
Updated minimum version to `0.0.12` :smile: